src/Components/ExternalSiteLink/ExternalSiteLink.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { View, Text, TouchableOpacity, Linking } from 'react-native';
+import IonIcon from 'react-native-vector-icons/Ionicons';
+import MaterialIcon from 'react-native-vector-icons/MaterialIcons';
+
+import styles from './styles';
+
+type ExternalLinkProps = {
+  text: string;
+  url: string;
+};
+
+export default function ExternalSiteLink({ text, url }: ExternalLinkProps) {
+  /**
+   * Routes user to the given url. Returns a Promise object:
+   * - If the user confirms the open dialog or the url automatically opens, the promise is resolved.
+   * - If the user cancels the open dialog or there are no registered applications for the url, the promise is rejected
+   */
+  async function openUrl() {
```

Comment:
Let's try to have this open in-app rather than direct to an external site. I'll help you do a bit of research but look into using `WebView` or anything you find fitting!